src/app/api/chat/stream/route.ts

```
@@ -0,0 +1,218 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { getServerSession } from 'next-auth/next';
+import { authConfig } from '~/server/auth/config';
+import { db } from '~/server/db';
+import { users, messages, usage, chats } from '~/server/db/schema';
+import { eq, sql as drizzleSql } from 'drizzle-orm';
+import { models } from '~/lib/model-config';
+import redis from '~/lib/redis';
+import type { Session } from 'next-auth';
+import { streamText, type CoreMessage } from 'ai';
+import { xai } from '@ai-sdk/xai';
+import { randomUUID } from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)